packages/components/src/core/MenuItem/index.stories.tsx

```diff
@@ -61,6 +61,316 @@ export const Default = {
   },
 };
 
+// Screenshot test
+
+const MULTI_SELECT_OPTIONS = [false, true];
+const DETAILS_OPTIONS = [undefined, "Details"];
+const COLUMN_OPTIONS = [undefined, "Column"];
+const ICON_OPTIONS = [undefined, "gear"];
+const SELECTED_OPTIONS = [false, true];
+const DISABLED_OPTIONS = [false, true];
+const PSEUDO_STATES = ["default", "hover", "active", "focus"];
+
+const ScreenshotTestDemo = (props: Args): JSX.Element => {
+  const DISPLAY_CONTENTS: React.CSSProperties = {
+    display: "contents",
+  };
+  const MID_LABEL: React.CSSProperties = {
+    borderStyle: "solid none none none",
+    gridColumn: "1 / 6",
+    justifySelf: "stretch",
+    paddingTop: 10,
+  };
+
+  // loop through all MULTI_SELECT_OPTIONS
+  return (
+    <>
+      {MULTI_SELECT_OPTIONS.map((isMultiSelect) => {
+        console.log(isMultiSelect);
+        return (
+          <MenuItemMultiSelect
+            isMultiSelect={isMultiSelect}
+            key={String(isMultiSelect)}
+          />
+        );
+      })}
+    </>
+  );
+
+  // loop through all DETAILS_OPTIONS + create headers for MULTI_SELECT_OPTIONS
+  function MenuItemMultiSelect({
+    isMultiSelect,
+  }: {
+    isMultiSelect: (typeof MULTI_SELECT_OPTIONS)[number];
+  }) {
+    const LEVEL_STYLE: React.CSSProperties = {
+      columnGap: "20px",
+      display: "inline-grid",
+      fontFamily: "sans-serif",
+      marginRight: "50px",
+    };
+    const LABEL_STYLE: React.CSSProperties = {
+      fontSize: "2em",
+      gridColumn: "1 / 6",
+      marginBottom: 0,
+    };
+
+    return (
+      <div style={LEVEL_STYLE}>
+        <p style={LABEL_STYLE}>
+          Multi-select: <b>{isMultiSelect ? "true" : "false"}</b>
+        </p>
+        {DETAILS_OPTIONS.map((details) => {
+          return (
+            <MenuItemDetails
+              isMultiSelect={isMultiSelect}
+              details={details}
```

Comment:
`details` is not rendering. When I try to pass the MenuItem some details text as a prop called `details`, it’s not rendering any of that text (looks just like a MenuItem without details). I can’t tell if there’s a different format I need to use (other than passing it as a prop called `details`) since details is also not included in the Default or Test stories, and I couldn’t find anything about details in its index.tsx file….
In the meantime, I can either still include details as a visual trait in the Chromatic screenshot test (it just won’t render properly until we know how to fix it), or I can remove it as a loop altogether (for now at least).